Sources/graphql-swift/codegen/schemaGenerator.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+struct GraphqlSchemaRequest: Codable {
+    let query: String
+}
+
+public struct SchemaTypeGenerator {
+    let client: GraphqlClient!
```

Comment:
No need to make this an implicitly unwrapped optional.

```suggestion
    let client: GraphqlClient
```

Reply:
thank you!